public/app/plugins/panel/singlestat/module.ts

```diff
@@ -198,6 +198,10 @@ class SingleStatCtrl extends MetricsPanelCtrl {
     this.setValueMapping(data);
   }
 
+   disableFontSizes() {
```

Comment:
This function name sounds like this does perform an action. maybe rename it to canChangeFontSize()